src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataSymbolMap.cs

```diff
@@ -16,6 +16,23 @@ namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
     internal class TaintedDataSymbolMap<TInfo> : IEquatable<TaintedDataSymbolMap<TInfo>?>
         where TInfo : ITaintedDataInfo
     {
+        private static bool TryResolveDependencies(TInfo info, WellKnownTypeProvider wellKnownTypeProvider)
+        {
+            if (info.DependencyFullTypeNames == null)
```

Comment:
```suggestion
            if (!info.DependencyFullTypeNames.IsEmpty)
```

Reply:
Needs to be inverted.

Reply:
Since it is used in foreach later I wonder if the check is needed at all.

Reply:
Oh, whoops, sorry for the incorrect `!`.

Good call, don't need the check with the foreach.